airflow/providers/cncf/kubernetes/CHANGELOG.rst

```diff
@@ -24,6 +24,20 @@
 Changelog
 ---------
 
+6.1.0
+.....
+
+Features
+~~~~~~~~
+
+* ``Add multiple exit code handling in skip logic for 'DockerOperator' and 'KubernetesPodOperator' (#30769)``
+* ``Skip KubernetesPodOperator task when it returns a provided exit code (#29000)``
+
+
+.. Below changes are excluded from the changelog. Move them to
+   appropriate section above if needed. Do not delete the lines(!):
+   * ``Deprecate 'skip_exit_code' in 'DockerOperator' and 'KubernetesPodOperator' (#30733)``
```

Comment:
I skip this PR from the change log because we don't really need to deprecate skip_exit_code
It was never released thus I will raise PR to remove the warning shortly

Reply:
https://github.com/apache/airflow/pull/30788